salt/utils/http.py

```diff
@@ -132,6 +132,7 @@ def query(url,
           handle=False,
           agent=USERAGENT,
           hide_fields=None,
+          raise_error=True,
```

Comment:
Does this default need to be set to `True`? Seems like that will trigger a behavior change here.